simulator/src/testbench.js

```
@@ -402,8 +402,12 @@ export function runAll(data, scope = globalScope) {
     let passedCases = 0;
 
     data.groups.forEach((group) => {
-        // for (const output of group.outputs) output.results = [];
-        group.outputs.forEach((output) => output.results = []);
+        const outputMap = new Map();
+        group.outputs.forEach((output) => {
+            output.results = [];
```

ESLint

[error] Assignment to property of function parameter 'output'.

(no-param-reassign)